frontend/src/metabase/dashboard/utils.js

```diff
@@ -103,6 +103,10 @@ export function getAllDashboardCards(dashboard) {
   return results;
 }
 
+export function isDatabaseActionEnabled(database) {
+  return database.settings?.["database-enable-actions"] ?? false;
+}
+
```

Comment:
nit: "is database action enabled" makes me think we're talking about some single action affecting the database somehow. I'm not sure, but maybe something like `hasDatabaseActionsEnabled` would make it a bit more clear? Please feel free to ignore that proposal though

Reply:
yep I agree with that as well - I keep thinking of database syncs now that you've mentioned it